source/assets/sass/vendor/typedoc/_highlight.css

```diff
@@ -1,3 +1,5 @@
+/* @@@ I don't think this file is used? */
+
```

Comment:
Yeah, it seems like this was made for a color theme switcher that hasn't been set up just yet @jgerigmeyer ?

Reply:
Agreed. Looks like it was added in https://github.com/sass/sass-site/commit/85a2eaa33d69b657131d32f0ae8d27673f6a4477, but never imported.